widget/view.controller.js

```diff
@@ -55,24 +54,25 @@
     $scope.feedRules = {
```

Comment:
These rules can't be hard coded in the wizard. These should be coming from key-store records.

Reply:
Also, widget should render all the rules present in the key-store record in a loop. Another enhancements could be that, rules json designed such a way that different ui component meta info can be part of the json and wizard can understand it to show in the wizard.

Reply:
will add this enhancement in next version